src/irr/video/COpenGLDescriptorSet.h

```diff
@@ -27,20 +27,20 @@ class COpenGLDescriptorSet : public IGPUDescriptorSet
     {
         struct SMultibindBuffers
         {
-            const GLuint* buffers;
-            const GLintptr* offsets;
-            const GLsizeiptr* sizes;
+            const GLuint* buffers = nullptr;
+            const GLintptr* offsets = nullptr;
+            const GLsizeiptr* sizes = nullptr;
         };
         struct SMultibindTextures
         {
-            const GLuint* textures;
-            const GLenum* targets;//for when ARB_multi_bind isn't there
-            const GLuint* samplers;
+            const GLuint* textures = nullptr;
+            const GLenum* targets = nullptr;//for when ARB_multi_bind isn't there
```

Comment:
I think we agreed that we need `ARB_multi_bind` from now on.

Reply:
yea it's just not used

Reply:
kill it